.changes/unreleased/Fixes-20231031-144837.yaml

```diff
@@ -0,0 +1,6 @@
+kind: Fixes
+body: Fix exception running empty seed file
```

Comment:
The PR and this changelog indicate we're solving multiple issues, can we add a second changelog to cover new data_type?

Reply:
This doesn't solve multiple issues, it's all the same issue. The problem with the empty seed file was happening because of the lack of support for the new Integer data type.

Reply:
Oh, you mean additional info in the existing changelog? Sure.

Reply:
yup! not questioning the PR just want to make sure our changelogs cover everything that's being changed